.github/workflows/release-core.yml

```diff
@@ -51,27 +48,18 @@ jobs:
       - name: Set Tegel user
         run: git config --global user.name "Tegel - Scania" && git config --global user.email "tegel.design.system@gmail.com"
 
-      - name: Core - CD into correct folder
-        run: cd packages/core
-
-      - name: Core - Bump version
-        run: npm version ${{ github.event.inputs.version }}
-
       - name: Core - Read Package.json Version
         id: version
         run: echo "version=$(jq -r .version package.json)" >> $GITHUB_OUTPUT
```

Comment:
Will this work? GPT-4 said this:

1. In the "Core - Read Package.json Version" step, you're writing to `$GITHUB_OUTPUT`, but this environment variable is not defined in GitHub Actions. You should use `::set-output name=version::$(jq -r .version package.json)` to set the output of a step. Here's the corrected step:

```
- name: Core - Read Package.json Version
  id: version
  run: echo "::set-output name=version::$(jq -r .version package.json)"
  working-directory: packages/core
```

🤷‍♂️

Reply:
They are deprecated: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/